atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -144,11 +144,6 @@ public int schemaMutationTimeoutMillis() {
     }
 
     @Value.Default
-    public int heartbeatTimePeriodMillis() {
```

Comment:
I saw the ADR and think it's well-written. It might be useful to additionally have some user-facing docs, maybe with a heading like "why we have a C* schema mutation lock and what to do if it stops heartbeating". A lot of the content might even be copy-pasted from the ADR, it's just that the ADRs aren't published as part of docs and are therefore harder for users to find / link to. 

Also tagging @rhero for context/input about how we should do docs. 

Having said all this, I want this bugfix PR to merge soon, and wouldn't block on the docs change.